### PR TITLE
add currency Tanzanian Shillings (TZS)

### PR DESCRIPTION
**Pull request by emma255**

### CODE REVIEW

File: src/Speller.php [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <?php
+
```
Comment (jurchiks) [inline src/Speller.php]: 1. Don't change the formatting!

File: src/Speller.php [hunk 2]
```diff
@@ -26,7 +27,8 @@ class Speller
 	const CURRENCY_RUSSIAN_ROUBLE = 'RUR';
 	const CURRENCY_US_DOLLAR = 'USD';
 	const CURRENCY_PL_ZLOTY = 'PLN';
-	
+	const CURRENCY_TANZANIAN_TZS = 'TZS';
```
Comment (jurchiks) [inline src/Speller.php]: 2. CURRENCY_TANZANIAN_SHILLING

File: src/languages/English.php [hunk 3]
```diff
@@ -117,11 +118,12 @@ public function getCurrencyNameMajor(int $amount, string $currency): string
 			Speller::CURRENCY_RUSSIAN_ROUBLE => ['ruble', 'rubles'],
 			Speller::CURRENCY_US_DOLLAR      => ['dollar', 'dollars'],
 			Speller::CURRENCY_PL_ZLOTY       => ['zloty', 'zlote'],
+			Speller::CURRENCY_TANZANIAN_TZS       => ['shilling', 'shillings'],
```
Comment (jurchiks) [inline src/languages/English.php]: 3. Try to find out the translation in other languages as well.
I can tell you that major currency translations for Latvian are `['šiliņš', 'šiliņi', 'šiliņu']`, for Russian `['шиллинг', 'шиллинга', 'шиллингов']`, and you can copy the cents from minor currency in those languages as well.

Comment (jurchiks) [reply src/languages/English.php]: 4. the keys => values must be aligned like they are for all the other values in the code.

File: src/Speller.php (hunk 1 above)
Comment (emma255) [reply src/Speller.php]: Thanks, SOLVED

File: src/Speller.php (hunk 2 above)
Comment (emma255) [reply src/Speller.php]: SOLVED

File: src/languages/English.php (hunk 3 above)
Comment (emma255) [reply src/languages/English.php]: Done

**Comment by emma255** [comment]
Hello sir, I worked on your comments. Please proceed.

Thanks.

**Comment by jurchiks** [comment]
Great, thank you very much!

**Comment by jurchiks** [comment]
FYI, when you make pull requests, it is good practice to make a feature branch first and not work on master. In this case it's fine if this was all you wanted to do, but when you (or your teammates, if any) work on more than one thing at once, you can't do all your work on master.
This is a good, simple article about how it all works: https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow

**Pull request merged**